pallets/hyperdrive-outgoing/src/tezos.rs

```diff
@@ -77,17 +83,30 @@ fn assign_payload_schema() -> &'static Micheline {
     static ASSIGN_PAYLOAD_SCHEMA: OnceBox<Micheline> = OnceBox::new();
```

Comment:
I think this could be simplified by just having `fn assign_payload_schema() -> Micheline { ... }`.

Reply:
that would recreate the schema on each use, I guess the rust compiler is not memoizing the result